src/subHeader.css

```diff
@@ -0,0 +1,77 @@
+
+/* sub header css start */
+.appSubHeader {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 50px 0px;
+  }
+  
+  .headerBold {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 60px 10px 10px 10px;
+    font-weight: 600 !important;
```

Comment:
Please why did you use `!important` here?
is it necessary?

Reply:
removed it